cmd/phpcs-server/main_test.go

```diff
@@ -135,6 +136,14 @@ func Test_main(t *testing.T) {
 				flagVisibility: &[]string{"public"}[0],
 			},
 		},
+		{
+			"Run Main - WP PHPCompatibility Flag set",
+			args{
+				messageChannel: make(chan message.Message, 1),
+				timeOut:        1,
+				flagWPRuleset:  &[]string{"./testdata/wordpress.xml"}[0],
```

Comment:
Why not just use the actual standard in place of the ruleset file and then we don't have to include it in the testdata directory?

Reply:
I don't think we need to test that the file works, it's assumed a ruleset can be passed to the cli because it's a feature of `phpcs`. What is the benefit of storing the ruleset in `testdata`?

Reply:
We don't need to...  unless we want coverage.
We store the file in `testdata` because that is where the tests have access to the file. As per a previous comment, its not so much what is in it, rather that it loads.  So we don't need to keep this in sync with the real file.

Reply:
Or we could just make it a ruleset that loads the PHPCompatibility standard to verify the file is used. Then no need to keep it in sync.